### DISCUSSION TITLE
How to automatically include .srt, .ssa or .ass files with the same name as the source video.

### DISCUSSION THREAD

**Discussion by quebecman** [RandomEngy/VidCoder#839] [General]
Hello,
  
  I was just wondering how this can be done.  Do we just have to DRAG and DROP the SRT and the VIDEO in the same folder?

Of course they need to be named the same.

Thanx.
Regards.

**Comment by RandomEngy** [comment]
Yes, it looks in the same folder as the video file.

**Comment by quebecman** [comment]
Thanx, I will give it a try for shure.  QA.
